src/test/java/teammates/test/cases/browsertests/FeedbackConstSumOptionQuestionUiTest.java

```diff
@@ -230,4 +232,31 @@ private void testUiConsistencyForNewQuestion() {
         feedbackEditPage.waitForConfirmationModalAndClickOk();
 
     }
+
+    /**
+     * Test that the constSumPoints* number fields gets disabled, if the corresponding radio button is not checked.
+     */
+    private void testConstSumPointsDisableAction() {
```

Comment:
Nitpicking: `testDisableConstSumPoints` instead?

Reply:
How about `testDisableConstSumPointsAction()` instead? To make it similar to `testAddQuestionAction()` etc tests?